pages/index.js

```diff
@@ -1,3 +1,21 @@
+import useSWR from "swr";
+import EventCard from "../components/EventCard";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: events, error } = useSWR("/api/events", fetcher);
+  if (error) return <h1>There was an error</h1>;
+  if (!events) return <h1>...Loading...</h1>;
+
+  events.sort((a, b) => new Date(a.date) - new Date(b.date));
+
+  return (
+    <>
```

Comment:
Where is your `main` element? 🙃 
```suggestion
    <main>
```